### PR TITLE
 Esthablishing Cat Tinder Read Functionality, working through User st…

### PR DESCRIPTION
**Pull request by XavierB91**
…ories, passing baton to partner

### CODE REVIEW

File: src/pages/CatIndex.js [hunk 1]
```diff
@@ -1,9 +1,44 @@
...
+    Enjoys: {cat.enjoys}
+    </CardText>
+    <Button>
+     Meow your match ! 🐈
+    </Button>
+  </CardBody>
+</Card>
+        )
+      })}
+    </main>
```
Comment (elysemontano) [inline src/pages/CatIndex.js]: Please clean up your indentation

**Review by elysemontano — CHANGES_REQUESTED** [review]
I don't see the show functionality in this PR and it is not checked on your Trello card.  This should all be grouped together in your Read functionality.  Also, please make sure to clean up indentation on you index page

**Review by XavierB91 — COMMENTED** [review]
Fixing indentation for visual styling

**Review by elysemontano — CHANGES_REQUESTED** [review]
Still not seeing show page setup in here.  This is part of your Read Functionality

**Pull request closed without merging**